genomicFileHandler/concat.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse, os, sys
+
+MY_DIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.append( MY_DIR )
+
+import genomic_file_handlers as genome
+
+def vcf(infileList, outfile):
+
+    with open(outfile, 'w') as vcfout:
+
+        headerWritten = False
+
+        for file_i in infileList:
```

Comment:
Another way is `for file_index, file_i in enumerate(infileList):` and then just have `headerWritten = file_index > 0`